src/main/java/com/jmb/springfactory/service/productionorder/ProductionOrderServiceImpl.java

```diff
@@ -2,22 +2,30 @@
 
 import static com.jmb.springfactory.service.UtilsService.logCreatedEntity;
 
+import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.function.Function;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Stream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)